doc/source/serve/tutorials/object-detection.md

```diff
@@ -0,0 +1,42 @@
+(serve-object-detection-tutorial)=
+
+# Serving Object Detection Model
+In this guide, we will show you how to build an object detection application with Ray Serve.
+
+For this example, we use [ultralytics/yolov5](https://github.com/ultralytics/yolov5) model and FastAPI to build the example. 
+
+Serve Code:
+```{literalinclude} ../doc_code/object_detection.py
+```
+
```

Comment:
Should we have a Prerequisites section at the top that says that Ray has been installed?